evergreen.yml

```diff
@@ -368,28 +368,29 @@ functions:
         set -eo pipefail
 
         rm -rf /data/mci/gennytoolchain*
+        rm -rf /data/mci/curator*
 
         export DEVELOPER_DIR="/Applications/Xcode10.1.app"
 
         ./scripts/lamp \
           -d=${distro|not-linux} \
           -DENABLE_JASPER:BOOL=ON \
-          -DGENNY_INSTALL_DIR:PATH=${workdir}/dist \
+          -DGENNY_INSTALL_DIR:PATH=${workdir}/src/dist \
```

Comment:
The genny script looks in `dist` in the root of the genny repo for the `genny_core` binary (since that's where it gets built by default). Note that the `src` directory mentioned above isn't the `src` directory inside the repo. `src` is the genny repository which seems kinda confusingly named

Reply:
> `src` is the genny repository itself

Right, because we set it to be `src` on line 376.

I think using workdir was wrong to start with. `PATH=$PWD/dist` is probably what we want to do here.